commit a1b2c3d4e5f6a1b2c3d4e5f6a1b2c3d4e5f6a1b2
Author: Sandook <sandook@example.com>
Date:   Mon Dec 22 00:00:00 2025 +0000

    Disable pyverbs build in rdma-core for Python 3.12 compatibility.
    
    The pyverbs Python bindings have Cython compatibility issues with
    Python 3.12. Since Sandook only uses the C libraries from rdma-core,
    we disable pyverbs to avoid build failures.

diff --git a/build/init_submodules.sh b/build/init_submodules.sh
--- a/build/init_submodules.sh
+++ b/build/init_submodules.sh
@@ -31,7 +31,7 @@ fi
 echo building RDMA-CORE
 cd rdma-core
 git -c user.name="x" -c user.email="x" am ../build/patches/rdma-core/*
-if ! EXTRA_CMAKE_FLAGS=-DENABLE_STATIC=1 MAKEFLAGS=-j$CORES ./build.sh; then
+if ! EXTRA_CMAKE_FLAGS="-DENABLE_STATIC=1 -DNO_PYVERBS=1" MAKEFLAGS=-j$CORES ./build.sh; then
   echo "Building rdma-core failed"
   echo "If you see \"Does not match the generator used previously\" try running \"make submodules-clean\" first"
   exit 1

